src/docker/orb.yml

```diff
@@ -430,6 +432,16 @@ jobs:
           else
             image_name="${DOCKER_NAMESPACE}/${PROJECT_NAME}"
           fi
+          if [ "${DOCKER_SERVER}" = "docker.pkg.github.com" ]; then
+              # hack: github need :owner/:repo_name/:image_name
+              # we have something like:
+              #   docker.pkg.github.com/ledgerhq/ledger-vault-api:develop
+              # we need:
+              #   docker.pkg.github.com/ledgerhq/ledger-vault-api/ledger-vault-api:develop"
+              export DOCKER_TAG=$(echo ${DOCKER_TAG} | sed 's%\([a-zA-Z0-9\.\-]*\)/\([a-zA-Z0-9\.\-]*\)/\([a-zA-Z0-9\.\-]*\):\([a-zA-Z0-9\.\-]*\)%\1/\2/\3/\3:\4%g')
+              echo DOCKER_TAG=${DOCKER_TAG} >> $BASH_ENV
```

Comment:
I'd suggest prefixing by `export`, I faced some curses in the past when not using it.